mappings/platform_file_centric.json

```diff
@@ -0,0 +1,319 @@
+            "type": "nested",
+            "properties": {
+              "id": {
+                "type": "keyword"
+              },
+              "treatment_type": {
+                "type": "keyword"
+              },
+              "response_to_therapy": {
+                "type": "keyword"
+              }
+            }
+          },
+          "specimens": {
+            "type": "nested",
+            "properties": {
+              "id": {
+                "type": "keyword",
+                "copy_to": ["file_autocomplete"]
+              },
+              "submitter_specimen_id": {
+                "type": "keyword",
+                "copy_to": ["file_autocomplete"]
+              },
+              "tumour_normal_designation": {
+                "type": "keyword"
+              },
+              "specimen_tissue_source": {
+                "type": "keyword"
+              },
+              "specimen_type": {
+                "type": "keyword"
+              },
+              "pathological_tumour_staging_system": {
+                "type": "keyword"
+              },
+              "pathological_t_category": {
+                "type": "keyword"
+              },
+              "pathological_n_category": {
+                "type": "keyword"
+              },
+              "pathological_m_category": {
+                "type": "keyword"
+              },
+              "pathological_stage_group": {
+                "type": "keyword"
+              },
+              "specimen_anatomic_location": {
+                "type": "keyword"
+              },
+              "tumour_histological_type": {
+                "type": "keyword"
+              },
+              "tumour_grading_system": {
+                "type": "keyword"
+              },
+              "tumour_grade": {
+                "type": "keyword"
+              },
+              "samples": {
+                "type": "nested",
+                "properties": {
+                  "id": {
+                    "type": "keyword",
+                    "copy_to": ["file_autocomplete"]
+                  },
+                  "submitter_sample_id": {
+                    "type": "keyword",
+                    "copy_to": ["file_autocomplete"]
+                  },
+                  "sample_type": {
+                    "type": "keyword"
+                  },
+                  "matched_normal_submitter_sample_id": {
+                    "type": "keyword"
+                  }
+                }
+              }
+            }
+          }
+        }
+      },
+      "repositories": {
+        "type": "nested",
+        "properties": {
+          "code": {
+            "type": "keyword"
+          },
+          "name": {
+            "type": "keyword"
+          },
+          "organization": {
+            "type": "keyword"
+          },
+          "country": {
+            "type": "keyword"
+          },
+          "url": {
+            "type": "keyword"
+          }
+        }
```

Comment:
Am I understanding correctly?
* In maestro_file_centric, there will be only 1 repository (the RDPC that generated the file).  That index doesn't need to be aware that other copies of the same file exist in another repository.
* But in platform_file_centric, all repositories with a copy of this file will be indexed.

Reply:
- `maestro_file_centric` built at RDPCs will have 1 repository
- `maestro_file_centric` built at DCC will have multiple repositories
- `plaftorm_file_centric` built at DCC will consume DCC `maestro_file_centric` and will have multiple repositories

Reply:
We have to develop a method/ETL/update to supplement the maestro file_centric with clinical data. the output of that is the platform_file_centric